sample/http-server.c

```diff
@@ -392,11 +392,15 @@ parse_opts(int argc, char **argv)
 }
 
 static void
-do_term(int sig, short events, void *arg)
```

Comment:
There is no actually signal support on windows, let's move related code under `#ifndef _WIN32`